Mathlib/Algebra/Lie/Sl2.lean

```diff
@@ -150,7 +150,7 @@ local notation "ψ" n => ((toEnd R L M f) ^ n) m
 set_option linter.unusedVariables false in
 @[nolint unusedArguments]
 lemma lie_f_pow_toEnd_f (P : HasPrimitiveVectorWith t m μ) (n : ℕ) :
-    ⁅f, ψ n⁆ = ψ (n + 1) := by
+    ⁅f, ψ n⁆ = ψ(n + 1) := by
```

Comment:
I think all these with `ψ` are wrong. It's essentially function application, so we use a space for that.
```suggestion
    ⁅f, ψ n⁆ = ψ (n + 1) := by
```